maintaining-and-tuning/monitoring.md

```diff
@@ -198,16 +224,160 @@ These instructions assume that GRR server and Prometheus are both up and running
-If you want to disable or remove an alert, go to the dashboard and the corresponding panel, and there you can remove the alerting rule.
-
-1. You can now use the dashboards. The dashboards can give a general overview over the main components of the GRR server and Fleetspeak servers, which can be utilized by the user to monitor different metrics of each component. Examples for such metrics can be found in the [examples above](#example-queries). Remember that the dashboards and alerts are flexible, and can be expanded or modified to adjust to your exact needs. Additional metrics can be used by exploring `http://<host>:<port>/metrics` for each component of GRR server (change the port according to the GRR server component you want) and Fleetspeak servers, and if you'd like to create your own [custom dashboards](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#create-a-dashboard), [panels](https://grafana.com/docs/grafana/latest/panels/add-a-panel/#add-a-panel) and [alerts](https://grafana.com/docs/grafana/latest/alerting/create-alerts/#create-alerts), make sure to go over the corresponding documentation in Grafana.
+1. Grafana is now set up and can be visited at `http://localhost:3000`.
+The username and password should be "admin"; please change it.
+If the Grafana UI doesn't show up, either the
+[installation of Grafana](https://grafana.com/docs/grafana/latest/installation/#install-grafana)
+or the [server run](https://grafana.com/docs/grafana/latest/installation/debian/#2-start-the-server)
+failed; make sure to check the official documentation.
+
+1. [Set up Prometheus as a data source](https://grafana.com/docs/grafana/latest/features/datasources/prometheus/#prometheus-data-source)
+for Grafana, so that Grafana can display the metrics from Prometheus.
+To do that, follow the
+[these instructions](https://grafana.com/docs/grafana/latest/features/datasources/add-a-data-source/#add-a-data-source):
+Click on Configuration -> Data Sources in the Grafana server UI, then click
+on the "Add data source" button. From the list of "time series databases", choose
+Prometheus and in the Name field type `grr-server` (for Fleetspeak, type `fleetspeak`).
+
+1. Grafana is set up and ready to show metrics scraped by Prometheus. You
+can start by either
+[creating your own dashboards](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#create-a-dashboard)
+or [importing exisiting dashboards](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard)
+into Grafana.
+You can choose to import sample dashboards from the
```

Comment:
I'd rephrase it along the lines of: "The next step would be to import dashboards provided with GRR. You can customize and expand them with additional graphs, if needed".
Basically, we have to present the shipped dashboards as a main option.